src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -156,7 +156,9 @@ describe('AuthenticatedAPIClient auth interface', () => {
   it('has method getAuthenticationState that returns authentication state when valid JWT cookie exists', () => {
     mockCookies.get.mockReturnValueOnce(encodedValidJwt);
     const result = client.getAuthenticationState();
-    expect(result.authentication.email).toEqual(validJwt.email);
+    expect(result.authentication.userId).toBeDefined();
+    expect(result.authentication.userId).toEqual(validJwt.user_id);
```

Comment:
FYI: Due to two issues, this was passing when neither of these were correct because they were both undefined.